Shared/en.lproj/Localizable.strings

```diff
@@ -116,7 +116,7 @@
 "Stop Sharing" = "Stop Sharing";
 
 /* No comment provided by engineer. */
-"Stop sharing after files have been sent (uncheck to allow downloading individual files)" = "Stop sharing after files have been sent (uncheck to allow downloading individual files)";
+"Stop sharing after files have been sent (uncheck to allow downloading individual files)" = "Stop sharing after files are sent. (Otherwise allows downloading individual files.)";
```

Comment:
```suggestion
"Stop sharing after files have been sent (uncheck to allow downloading individual files)" = "Stop sharing after all files are sent, instead of offering individual files with no time limit.)";
```
Guessing.

Reply:
This is definitely a sentence which needs rework, I give you that and it's going to be hard, because this flag does two things in one, wich are somewhat mutually dependent technically.

- It enables auto-stop of sharing, after files have been downloaded.
- To make that possible, we cannot allow the download of individual files anymore, because otherwise we couldn't detect the successful end of a download.

To be more clear in terms of UX about what's going on here, would most probably mean, that we split this feature up properly and have two checkboxes:

[] Allow download of individual files
[] Stop sharing after files are sent

If someone checks the later, the first will be unchecked and disabled or vice-versa.

However, there's 2 reasons which stand against this design:

1. Tradition: It's already done like it is in OnionShare desktop and doing it like this would also slightly change functionality.
2. Screen estate is scarce on mobile. (We support devices down to the iPhone SE, which is the screen size originally introduced with iPhone 5.)

Alternative would be doing it with radio-buttons. Alas, there's no native radio button UI element on iOS, therefore we would have to introduce that, which is somewhat icky.


Now to my critique of your suggestion:

> Stop sharing after *all* files

Well, adding this is somewhat superfluous, and also difficult in the sense, that we don't share individual files in that situation anymore, but one archive file instead. So technically we only share one file, but *all* content.

> files are sent

Grammatically, the correct time to use should be future perfect, at least to my limited understanding. The original does that. Your change would introduce incorrect grammar. It's definitely simpler and shorter, I give you that. I'm not into Simple English, so if you can argue, that that's Simple English rules, well, ok.

> instead of offering individual files with no time limit.

It feels kinda strange. It's a correct explanation to what happens, but it leaves the reader confused why this seemingly disconnected things are either-or. At least to me.

Let me have a try:

> Stop after files have been sent. (Disables download of individual files.)

How do you feel about that?

Reply:
I thought "Stop once the files are sent, instead of offering them individually." but then was confused as to whether it is the one (ZIP?) file that is being downloaded.

```suggestion
"Stop sharing after files have been sent (uncheck to allow downloading individual files)" = "Require downloading all files at once to stop sharing afterwards.";
```

Reply:
Ok, I'll change it to my suggestion.